py_gtfs_rt_ingestion/batch_files.py

```diff
@@ -7,32 +7,34 @@
 from collections.abc import Iterable
 from typing import NamedTuple
 
-from py_gtfs_rt_ingestion import batch_files, file_list_from_s3
+from py_gtfs_rt_ingestion import batch_files
+from py_gtfs_rt_ingestion import file_list_from_s3
 
 import logging
 logging.basicConfig(level=logging.INFO)
 
 DESCRIPTION = "Generate batches of json files that should be processed"
 
 class BatchArgs(NamedTuple):
-    input_file: str
     s3_prefix: str
+    s3_bucket: str
     filesize_threshold: int
-    output: str
```

Comment:
no longer doing input files or output files here.

leaving in an option to print out events if invoked without `pretty` but wouldn't be surprised to see that go at some point too.